tests/pbft_manager_test.cpp

```diff
@@ -181,6 +181,185 @@ void check_2tPlus1_validVotingPlayers_activePlayers_threshold(size_t committee_s
   }
 }
 
+// Test that after some amount of elapsed time will not continue soft voting for same value
+TEST_F(PbftManagerTest, terminate_soft_voting_pbft_block) {
```

Comment:
I think we should not design unit test like this. This is not simulate the real process in PBFT manager.
And I think we don't need this. Since terminate soft voted value because the value include a bogus anchor. So the unit test looks like testing same thing with terminate_bogus_dag_anchor

Reply:
@gaoqi8649  there is a fundamental difference between the two tests, in that in one case we have received the block (but its "bogus") and in the other we have never received the block.